src/server_manager/web_app/app.ts

```diff
@@ -870,8 +870,7 @@ export class App {
       view.isServerManaged = true;
       const host = selectedServer.getHost();
       view.monthlyCost = host.getMonthlyCost().usd;
-      view.monthlyOutboundTransferBytes =
-          host.getMonthlyOutboundTransferLimit().terabytes * (10 ** 12);
+      view.monthlyOutboundTransferBytes = host.getMonthlyOutboundTransferLimit().terabytes;
```

Comment:
There's a misalignment here. You can't assign terabytes to bytes. You need to convert first. Revert?

Reply:
Done